docs/source/index.md

```diff
@@ -0,0 +1,82 @@
+---
+hide-sidebar: true
+---
+
+# Python cookiecutter
+
+A tool to automatically create a Python project structure ready to
+release via GitHub and [PyPI](https://pypi.org/).
+
+::::{grid} 1 2 2 3
+:gutter: 3
+:class-container: sd-p-3
+
+:::{grid-item-card} {fas}`rocket;sd-text-primary` Get Started
+:link: get_started
+:link-type: doc
+:text-align: center
+
+Learn how to begin using python cookiecutter.
+:::
+
+:::{grid-item-card} {fas}`tools;sd-text-primary` Project Setup
+:link: project_setup
+:link-type: doc
+:text-align: center
+
+Installation && Setup.
+:::
+
+:::{grid-item-card} {fas}`book-open;sd-text-primary` Documentation
+:link: documentation
+:link-type: doc
+:text-align: center
+
+Build and publish project documentation
+:::
+
+:::{grid-item-card} {fas}`handshake-angle;sd-text-primary` Contributing
+:link: contributing
+:link-type: doc
+:text-align: center
+
+Guidelines on How to Contribute.
+:::
+
+:::{grid-item-card} {fas}`code-branch;sd-text-primary` Versioning
+:link: versioning
+:link-type: doc
+:text-align: center
+
+Semantic and Automated versioning.
+:::
+
+:::{grid-item-card} {fas}`circle-check;sd-text-primary` Pre Commits
+:link: pre_commits
+:link-type: doc
+:text-align: center
+
+Set up pre-commit hooks.
+:::
+::::
+
+:::{note}
+This tool is based on
+[cookiecutter-napari-plugin](https://github.com/napari/napari-plugin-template)
+and provides a solid starting point for your projects.
+:::
```

Comment:
Could you remove this, and put "python-cookiecutter is based on
[cookiecutter-napari-plugin](https://github.com/napari/napari-plugin-template)" into the footer?
```suggestion
:::{note}
This tool is based on
[cookiecutter-napari-plugin](https://github.com/napari/napari-plugin-template)
and provides a solid starting point for your projects.
:::
```